serverAddons/AAW/addons/AAW_Server/Revive/fn_healAction.sqf

```diff
@@ -18,11 +18,11 @@ private _iconIdle = "";
 private _iconProgress = "\A3\Ui_f\data\IGUI\Cfg\HoldActions\holdAction_revive_ca.paa";
 private _condition = {
     alive _target
-     && alive CLib_Player
-     && !(CLib_Player getVariable [QGVAR(isUnconscious), false])
-     && !(_target getVariable [QGVAR(isUnconscious), false])
-     && (_target distance CLib_Player < 3)
-     && (side group _target == side group CLib_Player)
+    && alive CLib_Player
```

Comment:
i think that 1 space bevor a newline && condition can improve readability because you know that it is sub order to the upper line